ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewHeaderFooterViewDemoController.swift

```diff
@@ -133,9 +133,11 @@ extension TableViewHeaderFooterViewDemoController {
                 }
                 footer?.setup(style: .footer, attributedTitle: title)
 
+#if os(iOS)
```

Comment:
Is this a Swift specific ifdef? can I use it in objective C?

Reply:
It appears so https://developer.apple.com/documentation/xcode/running-code-on-a-specific-version/

Reply:
Yep it's available in both! For anyone else seeing this:
```swift
#if os(iOS) // or os(visionOS)
// ...
#endif
```

```c
// Objective-C
#if TARGET_OS_IOS // or TARGET_OS_VISION
// ...
#endif
```